packages/ui/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { forwardRef, HTMLChakraProps, Radio as ChakraRadio } from '@chakra-ui/react';
+
+export type RadioSize = 'md' | 'lg';
+
+export interface RadioProps extends Omit<HTMLChakraProps<'div'>, 'onChange' | 'defaultChecked'> {
+  /**
+   * Determines radio's and text size.
+   */
+  size?: RadioSize;
+  /**
+   * Determines state of the radio.
+   */
+  checked?: boolean;
+  /**
+   * Determines if input is disabled.
+   */
+  disabled?: boolean;
```

Comment:
I think we should comment the code only when it's that complicated that it can't speak for itself